### DISCUSSION TITLE
Auto ignoring git submodules with `vcs`

### DISCUSSION THREAD

**Discussion by aboqasem** [biomejs/biome#2218] [Q&A]
Are there any plans to support ignoring submodules with vcs?

**Comment by ematipico** [comment]
I think nobody proposed an idea for it.

**Reply by aboqasem** [reply]
Is this discussion sufficient to put it on the plan, or do I create another one?

I may try to contribute if you think it's a good first issue

**Comment by arendjr** [comment]
For now you can just put the submodule folders in your ignore list, but I agree it would be convenient to exclude them automatically.

**Reply by aboqasem** [reply]
Yup this is what im currently doing
